examples/gno.land/r/gnoland/pages/page_contribute.gno

```diff
@@ -0,0 +1,91 @@
+
+- Identify the bounty you want to work on, and first start working in your local environment.
+- Once you have spent time on the code related to the bounty, we recommend submitting a 'draft' PR as soon as possible.
+    - The draft PR doesn't indicate that the bounty has been assigned to you, others are free to work on other draft PRs for the bounty.
+    - After submitting the 'draft' PR, continue working until you are ready to mark the PR as "ready for review".
+    - The core team will review the bounty PR submission, and determine if it qualifies for the bounty reward.
+- Ask for clarification early if an element on the DoD or implementation design is unclear.
+    - Aside from posting the issue as a bounty, keeping regular updates with the core team on the bounty being undertaken is key to being on the right track.
+    - The Definition of Done requires adhesion to the [contributing guidelines](https://github.com/gnolang/gno/blob/master/CONTRIBUTING.md); additionally, it is expected that any newly added code or functionality is properly documented, tested and covered, at least in 80% of added code.
+    - You're welcome to propose additional features and work on an issue should you envision a plausible expansion or change in scope. The core team may assign a bounty to the additional work, or change the bounty with respect to the changed scope.
+
+You may make your submission at any time; however we invite you to publish your draft PR very early in the development process. This will make your work public, so you can easily get help by the core team and other community members. Additionally, your work can be continued by other people should you get stuck or no longer be willing to work on the bounty. Likewise, you can continue the abandoned or stuck work that someone else worked on. Don't fear your work being "stolen": if a submission is the result of multiple people's efforts, we will look to split the bounty in a way that is fair and recognises each participant in creating the final outcome.
+
+Participants in the gno.land Bounty Program must meet the legal Terms and Conditions referenced [here](https://docs.google.com/document/d/1aXrZ6japdAykB5FLmHCCeBZTo-2tbZQHSQi79ITaTK0).
+
+### Bounty sizes
+
+Each bounty is associated with a size, to which corresponds the maximum compensation for the work involved on the bounty.
+
+In some cases, the work associated with a bounty may be outstanding. When that happens, the core team can decide to award up to 25% of the bounty's value to the recipient.
+
+The value of the bounty, aside from the material completion of the task, considers the involved time in managing the created pull request and iterating on feedback.
+
+
+t-shirt size | expected compensation
+-------------|-----------------------
+[XS]         | $ 500
+[S]          | $ 1000
+[M]          | $ 2000
+[L]          | $ 4000
+[XL]         | $ 8000
+_[XXL]_ \*   | $ 16000
+_[3XL]_ \*   | $ 32000
```

Comment:
it sounds like too rigid for me.

maybe change the expected compensation with a range?

for instance, I've the feeling that i'll want intermediary things

or mmaybe we keep this, but then we specify exceptions in the body of the issues themselves

we can try like this

Reply:
> it sounds like too rigid for me.
> 
> maybe change the expected compensation with a range?
> 
> for instance, I've the feeling that i'll want intermediary things
> 
> or mmaybe we keep this, but then we specify exceptions in the body of the issues themselves
> 
> we can try like this

We added that the bounty is a "maximum"; so if you feel that a value would be too little and you may want intermediary values; you can bump it up to the next one.

additionally, we added that for outstanding contributions the core team can add up to an additional 25%.

I'd say this is flexible enough already, no?